### DISCUSSION TITLE
Gerenciamento de estado com React Hooks

### DISCUSSION THREAD

**Discussion by Sergioamjr** [frontendbr/forum#1684] [Discussão]
Olá, devs. Ano passado eu escrevi [um artigo](https://medium.com/reactbrasil/substituindo-o-redux-pelo-context-api-react-hooks-a70e995daa1d) mostrando como eu substitui o Redux com os React Hooks useReducer e useContext no meu app, mantendo os conceitos de actions, types, reducers, middleware, mapStateToProps.

Compartilhei na comunidade e ele foi até foi publicado no Medium do React Brasil.

Eu sentia que podia melhorar o meu exemplo e o artigo e acabei reescrevendo meu código e também o artigo, deixando mais detalhado e organizado.

Gostaria muito de compartilhar com vocês, caso tenham curiosidade de ver como fiz ou queriam fazer o mesmo um dia.

Sugestões, comentários, críticas são sempre bem vindos.

https://www.sergiojunior.com.br/substituindo-redux-por-hooks/

**Comment by felipefialho** [comment]
Ficou foda @Sergioamjr, parabéns.

**Comment by joaolavoierfh** [comment]
Parabéns pelo artigo @Sergioamjr
